### PR TITLE
Added content Loader to blog Page

### LINKED ISSUE #3165: Content Loader required in Blog page too
**Actual Behaviour**

Screenshot 2019-12-29 at 2 57 24 PM

**Expected Behaviour**
Content Loader should be used

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by anishagg17**
Fixes #3165

Changes: Added content Loader to Blog page too.
these are responsive with device

Demo Link: https://pr-3166-fossasia-susi-web-chat.surge.sh

Screenshots of the change:
Hnet-image

### CODE REVIEW

**Comment by anishagg17** [comment]
@snitin315 @akshatnitd  Please review these changes

**Comment by snitin315** [comment]
For UI based changes we have to consider others opinions as well

**Comment by anishagg17** [comment]
> We already have a simple fetching animation for blog page. In my opinion as the blog page does not take much time to load the corresponding form of lazy loading is not suitable.

Sir , this content loading is used on home page too.
Thus my pr will make loading stuff uniform throughout the application.

**Comment by snitin315** [comment]
> Sir , this content loading is used on home page too.
> Thus my pr will make loading stuff uniform throughout the application.

The home page content takes much greater time to load , for uniformity please increase the radius and width of circle and rectangle respectively.

**Comment by anishagg17** [comment]
> > Sir , this content loading is used on home page too.
> > Thus my pr will make loading stuff uniform throughout the application.
> 
> The home page content takes much greater time to load , for uniformity please increase the radius and width of circle and rectangle respectively.

Sir, I have done with increased radius,width earlier 
It was not looking nice .  So i decreased width and radius

**Comment by anishagg17** [comment]
Screenshot 2019-12-30 at 8 22 32 PM


<hr/>

Screenshot 2019-12-30 at 8 23 45 PM

<hr/>


Screenshot 2019-12-30 at 8 24 40 PM

<hr/>
Sir ,Which one do you think is best 1st, 2nd or 3rd?

**Comment by snitin315** [comment]
> Screenshot 2019-12-30 at 8 22 32 PM
> 
this one looks good to me as per current blog page UI

**Comment by anishagg17** [comment]
Hnet-image (1)
I have now implemented these content loader in pr

File: src/components/About/Blog/BlogLoader.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  position: relative;
+
+  * {
+    position: relative;
+    left: 15%;
+    margin-top: 30px;
+  }
+`;
+
+const SingleLoader = () => (
```
Comment (akshatnitd) [inline src/components/About/Blog/BlogLoader.js]: Rename to `BlogCardLoader` or `BlogCardSkeleton`

Comment (anishagg17) [reply src/components/About/Blog/BlogLoader.js]: Done Sir!

**Comment by anishagg17** [comment]
sir can you please review this pull request too.  https://github.com/fossasia/susi.ai/pull/3157

**Pull request merged**